packages/core/stories/tooltip.stories.tsx

```diff
@@ -73,20 +98,67 @@ ScrollTooltip.args = {
   placement: "top",
 };
 
-export const ErrorTooltip: ComponentStory<typeof Tooltip> = Template.bind({});
```

Comment:
What's the benefit on not using templates?

Reply:
@joshwooding if you try putting all 3 tooltip stories into the docs page it crashes, this was to try and fix that (but it doesn't) - i believe its something to do with the tooltip hook, would you be able to take a look?